content/300-guides/300-upgrade-guides/800-upgrade-from-prisma-1/03-upgrading-the-prisma-layer.mdx

```diff
@@ -435,7 +435,7 @@ model Post {
 model Profile {
   id   String  @id @default(cuid())
   bio  String?
-  user String?
+  user String? @unique
```

Comment:
Are we sure that is actually the introspection result of the mentioend P1 schema? Just us changing our P4 implementation does not necessarily change the introspection result of such P1 schema and database. 

(The SQL is in a tab above, so we can test this)

Reply:
I think this is correct.The unique constraint is missing from the initial full schema sql, but as part of the update process the unique constraint is added manually. So if you introspect after having gone through the process this should be the result.

Reply:
Thanks, will leave as it is